src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -212,7 +216,31 @@ export const FilesTableRow = ({ file, removeFile, baseFieldName, showFileVersion
       <TableRow>
         <TableCell sx={{ pt: 0, pb: 0 }} colSpan={showFileVersion ? 6 : 5}>
           <Collapse in={openCollapsable}>
-            <Box sx={{ mt: '0.5rem', display: 'flex', justifyContent: 'space-evenly' }}>
+            <Box
+              sx={{
+                m: '1rem 1rem 0 1rem',
+                display: 'flex',
+                justifyContent: 'space-evenly',
+                gap: '1rem',
+                alignItems: 'start',
+              }}>
+              {user?.isCurator /*TODO: check for Publishing-Curator*/ && (
```

Comment:
avhenger av https://github.com/BIBSYSDEV/NVA-Frontend/pull/4979
```suggestion
              {user?.isPublishingCurator && (
```